internal/app/init/pkg/system/system_test.go

```diff
@@ -37,7 +38,8 @@ func (suite *SystemServicesSuite) TestStartStop() {
 		&MockService{name: "yolo"},
 	)
 	time.Sleep(10 * time.Millisecond)
-	system.Services(nil).Stop(
+	err := system.Services(nil).Stop(
 		context.TODO(), "yolo",
 	)
+	assert.Nil(suite.T(), err)
```

Comment:
you can also do `suite.Assert().NoError(err)` (assertions available on suite object)